### PR TITLE
Gobuild: Don't whitelist KBFS packages

### PR DESCRIPTION
**Pull request by gabriel**
Was whitelisting packages... we can use repo as is.

### CODE REVIEW

**Comment by gabriel** [comment]
@akalin

**Comment by akalin-keybase** [comment]
lgtm

File: react-native/gobuild.sh [hunk 1]
```diff
@@ -34,18 +34,15 @@ else
 fi
 
 go_kbfs_dir="$GOPATH/src/github.com/keybase/kbfs"
+mkdir -p "$go_kbfs_dir"
+
 if [ ! "$local_kbfs" = "1" ]; then
   echo "Getting KBFS (via go get)..."
-  go get github.com/keybase/kbfs/libkbfs
-  go get github.com/keybase/kbfs/fsrpc
+  go get github.com/keybase/kbfs
```
Comment (akalin-keybase) [inline react-native/gobuild.sh]: Looks like CI doesn't like this...you might need `go get github.com/keybase/kbfs/...`. Not sure why client works, but maybe local_client is always set?

Comment (gabriel) [inline react-native/gobuild.sh]: yeah.. fixing now

**Pull request merged**